docs/Modern-fetchQuery.md

```diff
@@ -0,0 +1,38 @@
+---
+id: fetch-query
+title: fetchQuery
+---
+
+You can use the `fetchQuery` function to imperatively make GraphQL Requests. This is useful for cases where you want to make requests outside of React but still utilize the Relay store and network layer.
+
+```javascript
+import {fetchQuery, graphql} from 'relay-runtime';
+
+const query = graphql`
+  query ExampleQuery($pageID: ID!) {
+    page(id: $pageID) {
+      name
+    }
+  }
+`;
+
+const variables = {
+  pageID: '110798995619330',
+};
+
+fetchQuery(environment, query, variables)
+  .then(data => {
+    // access the graphql response
```

Comment:
Does this contain just the fulfilled data or does this object consist of `{ data, errors }` ?

Reply:
just the fulfilled `data`. As far as I can tell, there's no way to get the `errors` from this API.

Reply:
Would it throw?

Reply:
You can have both fulfilled data and errors, so I doubt that.

Reply:
The promise is rejected only if the GraphQL response returns `data: null`, similar to what's described in https://github.com/facebook/relay/issues/1913.

Reply:
Ah right, makes sense.